packages/kbn-es/src/utils/docker.ts

```diff
@@ -492,24 +491,22 @@ export async function setupServerlessVolumes(log: ToolingLog, options: Serverles
     volumeCmds.push(...fileCmds);
   }
 
-  if (ssl) {
-    const essResources = ESS_RESOURCES_PATHS.reduce<string[]>((acc, path) => {
-      acc.push('--volume', `${path}:${ESS_CONFIG_PATH}${basename(path)}`);
+  const essResources = ESS_RESOURCES_PATHS.reduce<string[]>((acc, path) => {
+    acc.push('--volume', `${path}:${ESS_CONFIG_PATH}${basename(path)}`);
 
-      return acc;
-    }, []);
+    return acc;
+  }, []);
 
-    volumeCmds.push(
-      ...getESp12Volume(),
-      ...essResources,
+  volumeCmds.push(
+    ...getESp12Volume(),
+    ...essResources,
 
-      '--volume',
-      `${ESS_SECRETS_PATH}:${ESS_CONFIG_PATH}secrets/secrets.json:z`,
+    '--volume',
+    `${ssl ? ESS_SECRETS_SSL_PATH : ESS_SECRETS_PATH}:${ESS_CONFIG_PATH}secrets/secrets.json:z`,
```

Comment:
ES Serverless will error and fail on startup if we have unused secrets, so for now we duplicate files somewhat. If this becomes a burden to maintain in the future, We will dynamically combine and write `secrets.json` as needed and using `.gitignore` to avoid constant git diff.